.github/ISSUE_TEMPLATE/api_bug.md

```diff
@@ -105,13 +105,13 @@ Stack trace:
 # Checklist
 
 Check all boxes that apply to this issue :
- - [ ] Bug description is provided
- - [ ] Expected comportment is provided
- - [ ] Request Path is provided
- - [ ] Request Method is provided
- - [ ] Request Headers are provided
- - [ ] Request Body is provided (for POST and PATCH requests only)
- - [ ] Response Status is provided
- - [ ] Response Headers are provided
- - [ ] Response Body is provided
+ - [ ] bug description is provided
+ - [ ] expected comportment is provided
+ - [ ] request Path is provided
```

Comment:
This change was meant to be the other way around. Should be: 
`Request path is provided` etc. beginning of the itemization is capitalized like in the beginning of a sentence and the following starts small.